api/v1beta1/flowcollector_types.go

```diff
@@ -234,9 +234,7 @@ type FlowCollectorEBPF struct {
 	// - `PacketDrop`: enable the packets drop flows logging feature. This feature requires mounting
 	// the kernel debug filesystem, so the eBPF pod has to run as privileged.
 	// If the `spec.agent.eBPF.privileged` parameter is not set, an error is reported.<br>
-	// - `DNSTracking`: enable the DNS tracking feature. This feature requires mounting
-	// the kernel debug filesystem hence the eBPF pod has to run as privileged.
-	// If the `spec.agent.eBPF.privileged` parameter is not set, an error is reported.<br>
+	// - `DNSTracking`: enable the DNS tracking feature.<br>
```

Comment:
we need to probably say here to get proper DNS tracking records we need to run with sample: 1